### PR TITLE
Support vignette-poz URLs

### PR DESCRIPTION
**Pull request by rogatty**
Support URLs like http://vignette-poz.wikia-dev.com/thelastofus/9/99/Robert.png/revision/latest

### CODE REVIEW

File: src/vignette.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ interface ThumbnailOptions {
 }
 
 class Vignette {
-	private static imagePathRegExp: RegExp = /\/\/vignette\d?\.wikia/;
+	private static imagePathRegExp: RegExp = /\/\/vignette(\d|-poz)?\.wikia/;
```
Comment (hakubo) [inline src/vignette.ts]: How about -sf urls?

Comment (rogatty) [inline src/vignette.ts]: I've searched through chef-repo and there is nothing like that. SF uses `vignette.wikia-dev.com`.

**Comment by drsnyder** [comment]
Please provide a [description](https://github.com/Wikia/guidelines-internal/blob/master/Process/PullRequests.md#guidelines) in your pull requests.

File: dist/vignette.cjs.js [hunk 2]
```diff
@@ -230,7 +230,7 @@ var Vignette = (function () {
         }
         return modeParameters.join('/');
     };
-    Vignette.imagePathRegExp = /\/\/vignette\d?\.wikia/;
+    Vignette.imagePathRegExp = /\/\/vignette(\d|-poz)?\.wikia/;
     Vignette.domainRegExp = /(wikia-dev.com|wikia.nocookie.net)/;
     Vignette.legacyPathRegExp = /(wikia-dev.com|wikia.nocookie.net)\/__cb[\d]+\/.*$/;
```
Comment (drsnyder) [inline dist/vignette.cjs.js]: Can we inject this domains as part of the setup? It seems brittle (and very Wikia specific) to have the configuration so tightly coupled here.

Comment (hakubo) [inline dist/vignette.cjs.js]: I agree with you, but can you create a separate ticket for this?

**Comment by mklucsarits** [comment]
LGTM

**Pull request merged**